app/(aux)/prayerCircleInfo.js

```diff
@@ -0,0 +1,565 @@
+										>							
+											<StyledImage
+												className='rounded-xl'
+												style={{ width: 60, height: 60 }}
+												source={require('../../assets/devs/alex.jpg')}
+											/>
+											<StyledText className={`font-[600] text-offwhite text-[20px]`}>
+												Alex Muresan
+											</StyledText>
+										</StyledPressable>							
+									</StyledView>
+
+									<StyledView className='flex left-4'>		
+										<StyledPressable
+											className='flex-col w-auto items-center justify-center'
+											onPress={() => (router.push('https://www.linkedin.com/in/andrew-roberti-4724a8263/'))}
+										>							
+											<StyledImage
+												className='rounded-xl'
+												style={{ width: 60, height: 60 }}
+												source={require('../../assets/devs/andrew.jpg')}
+											/>
+											<StyledText className={`font-[600] text-offwhite text-[20px]`}>
+												Andrew Roberti
+											</StyledText>
+										</StyledPressable>					
+									</StyledView>
+								</StyledView>
+								
+								<StyledView className='z-10 flex-row justify-evenly'>
+									<StyledView className='flex'>			
+										<StyledPressable
+											className='flex-col w-auto items-center justify-center'
+											onPress={() => (router.push('https://www.linkedin.com/in/nason-allen/'))}
+										>							
+											<StyledImage
+												className='rounded-xl'
+												style={{ width: 60, height: 60 }}
+												source={require('../../assets/devs/nason.jpg')}
+											/>
+											<StyledText className={`font-[600] text-offwhite text-[20px]`}>
+												Nason Allen
+											</StyledText>
+										</StyledPressable>							
+									</StyledView>
+
+									<StyledView className='flex'>		
+										<StyledPressable
+											className='flex-col w-auto items-center justify-center'
+											onPress={() => (router.push('https://www.linkedin.com/in/justin-ayres-25576921b/'))}
+										>							
+											<StyledImage
+												className='rounded-xl'
+												style={{ width: 60, height: 60 }}
+												source={require('../../assets/devs/justin.jpg')}
+											/>
+											<StyledText className={`font-[600] text-offwhite text-[20px]`}>
+												Justin Ayres
+											</StyledText>
+										</StyledPressable>							
+									</StyledView>
+								</StyledView>
+							
+								<StyledImage
+									className='z-0 absolute top-2 self-center justify-center w-[260px] h-[260px]'															
+									source={require('../../assets/spiral/spiral.png')}
+								/>
+
+							</StyledView>
+						</StyledView> */}
+
+						<View className='relative pb-[75px]'></View>
+					</StyledView>
+				</ScrollView>
+
+				<StyledGradient
+					pointerEvents='none'
+					start={{ x: 0, y: 0.1 }}
+					end={{ x: 0, y: 1 }}
+					style={{ height: 120 }}
+					className='absolute w-screen'
+					colors={['#121212ee', 'transparent']}
+				/>
+				<StyledView
```

Comment:
I think we should copy the way that circleSettings does, with the text centered. on iOS it just looks a little off. i don't think that the terms of service should go at the top there, rather i think they should go as another element in the scrollview